packages/gestalt/src/ButtonToggle.tsx

```diff
@@ -44,6 +69,13 @@ type Props = {
    * Indicates if ButtonToggle is disabled. Disabled ButtonToggles are inactive and cannot be interacted with. See the [state variant](https://gestalt.pinterest.systems/web/buttontoggle#State) for details on proper usage.
    */
   disabled?: boolean;
+  /**
+   * An icon displayed above the text to illustrate the meaning of the option selected by the ButtonToggle.
+   */
+  graphicIcon?:
+    | { image: ReactElement<typeof Image> }
+    | { avatar: ReactElement<typeof Avatar> }
```

Comment:
i think we can keep it for 

`    | { image: ReactElement<typeof Image> }
`

for now. we can add later on to icon and avatar. it doesnt come in the specs

Reply:
DONE